cratedb_toolkit/io/iceberg.py

```
@@ -0,0 +1,121 @@
+    engine = sa.create_engine(str(cratedb_url))
+
+    pl.Config.set_streaming_chunk_size(CHUNK_SIZE)
+    table = iceberg_address.load_table()
+
+    # This conversion to pandas is zero-copy,
+    # so we can utilize their SQL utils for free.
+    # https://github.com/pola-rs/polars/issues/7852
+    # Note: This code also uses the most efficient `insert_bulk` method with CrateDB.
+    # https://cratedb.com/docs/sqlalchemy-cratedb/dataframe.html#efficient-insert-operations-with-pandas
+    table.collect(streaming=True).to_pandas().to_sql(
+        name=cratedb_table.table, schema=cratedb_table.schema, con=engine, if_exists="replace", index=False, chunksize=CHUNK_SIZE, method=insert_bulk,
```

Ruff (0.11.9)

Line too long (150 > 120)

(E501)